.rubocop_todo.yml

```diff
@@ -292,6 +284,13 @@ Style/IndentHash:
   Enabled: false
 
 # Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: Width.
+Style/IndentationWidth:
+  Exclude:
+    - 'lib/prison_schedule.rb'
```

Comment:
Where's this come from? Looks like you've got [a 1-space indent](#diff-6bff082cf145ad1fbaeecb4fd4df90f5R5):

```ruby
  def initialize(prison)
   @prison = prison
  end
```